### PR TITLE
Modified Readme

### PR DESCRIPTION
**Pull request by nschnitzer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,68 @@
...
+  },
+  "InteractionTarget": {},
+  "InteractionScreen": {
+    "screen": "core:containerScreen"
+  },
+  "Network": {
+  }
+}
+```
+where itemSlots should be filled with 0s as it is empty. "privateToOwner" details whether or not other users have access to the bookshelf. "PlaySoundAction" contains the sound played for when the bookshelf is opened. "InteractionScreen" contains information of how the bookshelf should be interacted with.
```
Comment (llvieira) [inline README.md]: Please use some markdown symbols to format the text. For example "privateToOwner" could be `"privateToOwner"` (same for any piece of code/json). It makes the whole text way more readable.

**Review by llvieira — CHANGES_REQUESTED** [review]
Some minor tweaks suggested. :+1:

File: README.md [hunk 2]
```diff
@@ -1,6 +1,68 @@
...
-- Quill - https://openclipart.org/detail/262818/vintage-feather-inkwell
\ No newline at end of file
+The Following Types of Components Can Be Made:
+* `Book`
+* `Bookcase`
+
+
+# Contribution
+
+Fork and clone this repository locally
```
Comment (pollend) [inline README.md]: we tend not to clone the module repository directly into the terasology workspace. there is some setup that gradle and groovyw will do when you grab a module. there is some documentation in the wiki that explains this already: https://github.com/MovingBlocks/Terasology/wiki/Developing-Modules

**Review by pollend — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (nschnitzer) [reply README.md]: This contradicts the examples you provided. [Ex1](https://github.com/Terasology/Potions) [Ex2](https://github.com/Terasology/Cooking) .

Comment (Cervator) [reply README.md]: Oh that's a good catch. Probably those were introduced, not caught at the time, then have spread around as poor examples. Feel free to fix that up for any other modules you notice it for. Not really a separate task but if you do it and highlight that to us that counts for intangible Cool Points to possibly get put to use on other stuff in the future :-)

Comment (nschnitzer) [reply README.md]: That sounds great. I'll do it when I get some time!

**Pull request merged**